src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -35,7 +37,7 @@ export plant_absorbed_pfd,
 Returns the constant leaf angle distribution value for the given G function.
 Takes in an arbitrary value for the solar zenith angle, which is not used.
 """
-function compute_G(G::ConstantGFunction{FT}, _::FT) where {FT}
+function compute_G(G::ConstantGFunction, _::FT) where {FT}
```

Comment:
Im not sure this is correct (also below). When we call this, we pass it theta_s as a Field, not as type FT. Also, FT is not needed here, in typing the arguments or in the where statement